astropy/modeling/core.py

```diff
@@ -154,9 +154,9 @@ def __init__(cls, name, bases, members, **kwds):
         pdict = {}
         for base in bases:
             for tbase in base.__mro__:
```

Comment:
Can this be compressed into another loop comprehension?

Reply:
looking at it more carefully I don't think it should; I think the more logical way to express what's going on here is actually:
```python
        pdict = {}
        for base in bases:
            for tbase in base.__mro__:
                if issubclass(tbase, Model):
                    pdict |= cls._parameters_
                    break
```
(unless `cls._parameters_` is somehow allowed to mutate in between iterations but that would seem extremely fragile)